package.json

```diff
@@ -23,14 +23,9 @@
     "generate-icons": "ts-node ./src/generate.ts"
   },
   "sideEffect": false,
-  "types": "./dist/index.d.ts",
   "main": "./dist/index.js",
-  "exports": {
-    ".": {
-      "require": "./dist/index.js",
```

Comment:
Does that mean that we don't provide cjs exports anymore?

Reply:
I found a couple of places where it says that setting `"main": "./dist/index.js",` is enough for cjs. But I might be wrong. What would be your recommendation?

Reply:
I am not that experienced with this kind of setting. @ndelangen is the best person to ask.

Reply:
On what basis are we making these changes? What's the rationale? Do we know what effects these changes will have and why?

I'd expect a PR description detailing what the problem was, how the code change fixes it and potentially the alternatives and potential alternatives.

This is the first time for me looking at this codebase as far as I can remember, so I can't determine if this change is ok or now.

- removing `cjs` exports seems/feels wrong
- adding the "pure"-comments in source seems wrong too.
But maybe there are valid reasons for doing this. I do not have enough context.

I do not trust the screenshots of a website I do not know the details of.
I was told it's running esbuild, but it's extremely dependent on how esbuil is configured, as well as other factors in our toolchain if the code is actually treeshaken in the end or not.
I'd like to see the actual tree-shaken code when consumed in the manager itself, and inside a manager-entry.

Reply:
@ndelangen We are not using the library anywhere right now. The question is more about is the best way to export the library in `package.json`. I made this change in `package.json` because it was recommended in a couple of articles I found but I'm definitely not an expert in that field.

Reply:
I guess I'm not sure what's the difference between:
```json
"main": "./dist/index.js",
"module": "./dist/index.mjs",
```

and

```json
"main": "./dist/index.js",
"exports": {
  ".": {
    "require": "./dist/index.js",
    "import": "./dist/index.mjs"
  }
}
```

In the later, it seem we still need `"main"`. I can't find a clear answer on the web.

Reply:
By the way @ndelangen I brought back the change.

Reply:
> Adding the pure comment is a compromise we found with @JReinhold and @valentinpalkovic to make sure forward Ref is considered pure. This PR for more context: https://github.com/evanw/esbuild/issues/2749

Thank you for that context & link, I agree that it makes sense then.